ddp-workspace/projects/toolkit/src/lib/components/activity/activity.component.ts

```diff
@@ -10,7 +10,8 @@ import { ActivityResponse } from 'ddp-sdk';
         <toolkit-header [showButtons]="false"
                         [stickySubtitle]="stickySubtitle">
         </toolkit-header>
-        <ddp-activity [studyGuid]="studyGuid"
+        <ddp-activity *ngIf="id"
+                      [studyGuid]="studyGuid"
```

Comment:
Could we rename instance variable "id" to "instanceGuid"? (Not sure how we ended up naming the param id, but at least it is used consistently)

Reply:
Sure. 
Done at 43f7e4f72aa16edc94f25ca0c02ccbecbe2cbffa